app/azure/rest/index.js

```diff
@@ -0,0 +1,13 @@
+import angular from 'angular';
+
+import { ContainerGroup } from './container-group';
+import { Provider } from './provider';
+import { ResourceGroup } from './resource-group';
+import { Subscription } from './subscription';
+
+export default angular
+  .module('portainer.azure.rest', [])
+  .service('ContainerGroup', ContainerGroup)
+  .service('Provider', Provider)
+  .service('ResourceGroup', ResourceGroup)
+  .service('Subscription', Subscription).name;
```

Comment:
Still using `.service()` instead of `.factory()` ; as stated in my previous review we need to apply it to all index files of the modyle that are using service

Reply:
Up on this.

As [$resource](https://docs.angularjs.org/api/ngResource/service/$resource) is a [factory](https://docs.angularjs.org/guide/providers#factory-recipe) (e.g. constructor function that retuns an object), not a [service](https://docs.angularjs.org/guide/providers#service-recipe), and we are basically returning `$resource` in our "REST services" we should register them as `.factory()`.

Our services (`app/azure/services/azure.service.js` for example) are written in a factory way (constructor fucn returning literal object) but they could conceptually be classes (so could be registered as `.service()`), using `.factory()` for our services in this situation is correct.

Great in-depth explanation can be found [here](https://blog.thoughtram.io/angular/2015/07/07/service-vs-factory-once-and-for-all.html).

Reply:
in the explanation he explicitly says that when services return an object there is no difference. guidelines were talking about the pattern (an not the angular resource).

Made the change to make it more explicit